next.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
why using this file? Though env file is there?

Reply:
I wasn't taking anything from env files that's why